server/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Created by https://www.gitignore.io/api/django
+
+### Django ###
+*.log
+*.pot
+*.pyc
+__pycache__/
+local_settings.py
+db.sqlite3
+media
+
+# If your build process includes running collectstatic, then you probably don't need or want to include staticfiles/
+# in your Git repository. Update and uncomment the following line accordingly.
+# <django-project-name>/staticfiles/
+
+
+# End of https://www.gitignore.io/api/django
+docker-compose.yaml
+docker-compose.yml
```

Comment:
Neither should this. This. .gitigtore could just be merged with the root .gitignore anyway.